actions/notification_actions.jsx

```diff
@@ -98,8 +101,10 @@ export function sendDesktopNotification(post, msgProps) {
         notifyText = notifyText.substring(0, 49) + '...';
     }
 
+    const strippedMarkdownNotifyText = formatWithRenderer(notifyText, new RemoveMarkdown());
```

Comment:
Since the `RemoveMarkdown` instance is invariant based on the message, can we create a constant for it and not create a new instance every time?